libpod/image/image.go

```diff
@@ -115,6 +115,17 @@ func (ir *Runtime) NewFromLocal(name string) (*Image, error) {
 	return &image, nil
 }
 
+// NewNoExist creates a new image object without attempting to pull the
+// image or resolve it in any way
+func (ir *Runtime) NewNoExist(name string) *Image {
```

Comment:
Can this and ToStorageReference go away now that we aren't using it?

Reply:
removed